flink-runtime/src/main/java/org/apache/flink/runtime/mailbox/FutureTaskWithException.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.flink.runtime.concurrent;
+package org.apache.flink.runtime.mailbox;
```

Comment:
Why did you move this one? As long as it stays in `flink-runtime` I think it makes more sense to have it in the same package as for example `FutureTask`.

Reply:
Yes, you are right. This one does not need to move place as it stays in `flink-runtime`.

It was mistakenly moved when I was trying out different movement places (making all to-be-moved classes in the same package help trying-out.

I moved it back to its original place.